packages/replay/src/index.ts

```diff
@@ -180,12 +180,14 @@ export class Replay implements Integration {
 
     // TODO(deprecated): Maintain backwards compatibility for alpha users
     if (usingDeprecatedCaptureOnlyOnError) {
+      // eslint-disable-next-line no-console
       console.warn(
         '[@sentry/replay]: The `captureOnlyOnError` option is deprecated! Please configure `errorSampleRate` instead.',
       );
     }
 
     if (usingDeprecatedReplaysSamplingRate) {
+      // eslint-disable-next-line no-console
       console.warn(
         '[@sentry/replay]: The `replaysSamplingRate` option is deprecated! Please configure `sessionSampleRate` instead.',
       );
```

Comment:
I think the change is food for now but WDYT, should we think about removing these deprecated options? We're still in alpha and soon bumping to 7.x so it might be wort a thought.

Reply:
Yes, I would remove them in a separate PR. I think it makes sense to put this into the 0.x --> 7.x changelog, as people will be most likely to actually read that!

Reply:
Yes, separate PR sounds good to me. Let's just make sure to keep the Replay team informed here.